src/main/java/com/bookstore/bookstoreapi/data/configuration/CreateABook.java

```diff
@@ -0,0 +1,86 @@
+package src.main.java.com.bookstore.bookstoreapi.data.configuration;
+
+import java.util.ArrayList;
+import java.util.List;
+import java.util.Scanner;
+import src.main.java.com.bookstore.bookstoreapi.data.entity.BookDetails;
+
+public class CreateABook {
```

Comment:
Please refer to my configuration class (Right after you edit the `Book.java` file.) to make sure you are able to create a book in the database not a local variable. This way you will not be able to make a API request, because this book does not exist.